README.adoc

```diff
@@ -343,6 +344,14 @@ The following table documents the patches we've made to PhantomJS and tracks the
 |{icon-exclamation-dim} todo
 |===
 
+=== WebSlides
```

Comment:
I would suggest to move that section within the `Commands` paragraph, at the bottom.